### PR TITLE
adds FAQ section, Real World Example section and answer to first question

### PR DESCRIPTION
**Pull request by nelsonic**
see: https://github.com/ideaq/hapi-auth-jwt2/issues/32 for discussion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,7 +183,25 @@ If you prefer *not* to use any of these verifyOptions simply
...
+## Frequently Asked Questions (FAQ)
+
+1. Do I need to include **jsonwebtoken** in my project? asked in  [hapi-auth-jwt2/issues/32](https://github.com/ideaq/hapi-auth-jwt2/issues/32)  
+**Q**: Must I include the **jsonwebtoken** package in my project
+[given that **hapi-auth-jwt2** plugin already includes it] ?  
+**A**: Yes, you need to *manually* install the **jsonwebtoken**
+node module from NPM with `npm install jsonwebtoken --save` if you want to ***sign*** JWTs in your app.  
+Even though **hapi-auth-jwt2** includes it
+as a **dependency** your app does not know where to find it in the **node_modules** tree for your project.  
+unless you include it via ***relative path*** e.g:
```
Comment (iteles) [inline README.md]: Typo: this should be either a capital 'U' or not after a full stop

File: README.md [hunk 2]
```diff
@@ -128,10 +128,27 @@ That's it.
...
 If you would like to see a "***real world example***" of this plugin in use
 in a ***production*** web app (API)
 please see: https://github.com/ideaq/time/tree/master/api/lib
 
++ **app.js** ***registering*** the **hapi-auth-jw2 plugin**:
+[app.js#L13](https://github.com/ideaq/time/blob/0a5ec8711840528a4960c388825fb883fabddd76/app.js#L13)
++ telling app.js where to find our **validateFunc**tion:
+[app.js#L21](https://github.com/ideaq/time/blob/0a5ec8711840528a4960c388825fb883fabddd76/app.js#L21)
++ **validateFunc**tion (how we check the JWT is still valid):
+[api/lib/auth_jwt_validate.js](https://github.com/ideaq/time/blob/0a5ec8711840528a4960c388825fb883fabddd76/api/lib/auth_jwt_validate.js) look up the person's session in our ElasticSearch Database
```
Comment (iteles) [inline README.md]: Suggestion: looks with an 's'? Or is this an action from the user?

Comment (nelsonic) [inline README.md]: look**s** would be better. I have amended.

File: README.md (hunk 1 above)
Comment (nelsonic) [inline README.md]: What line are you referring to...? :-|

Comment (nelsonic) [inline README.md]: Got it. line 199. corrected. please specify the line number next time. thanks!

Comment (iteles) [inline README.md]: Sorry, thought that the point of inline comments was that it would tell you which line - seems like it doesn't!

**Pull request merged**